py-polars/tests/test_lazy.py

```diff
@@ -482,19 +614,40 @@ def test_rolling(fruits_cars: pl.DataFrame) -> None:
     out = df.select(
         [
             pl.col("A").rolling_min(3, min_periods=1).alias("1"),
+            pl.col("A").rolling_min(3).alias("1b"),
             pl.col("A").rolling_mean(3, min_periods=1).alias("2"),
+            pl.col("A").rolling_mean(3).alias("2b"),
             pl.col("A").rolling_max(3, min_periods=1).alias("3"),
+            pl.col("A").rolling_max(3).alias("3b"),
             pl.col("A").rolling_sum(3, min_periods=1).alias("4"),
+            pl.col("A").rolling_sum(3).alias("4b"),
+            # below we use .round purely for the ability to do .frame_equal()
+            pl.col("A").rolling_std(3, min_periods=1).round(decimals=4).alias("std"),
+            pl.col("A").rolling_std(3).alias("std2"),
+            pl.col("A").rolling_var(3, min_periods=1).round(decimals=4).alias("var"),
+            pl.col("A").rolling_var(3).alias("var2"),
         ]
     )
 
+    # TODO: rolling_std & rolling_var return nan instead of null if it cant compute
```

Comment:
I think we would want nulls, regardless of whether the windows is too short per `min_periods` or because we cant compute a variance on just a single data point. Left it as a TODO here.